src/scripts/config.js

```diff
@@ -1,6 +1,9 @@
 (function () {
   'use strict';
 
+  var coachBasePath = '//openstax.github.io/concept-coach/';
```

Comment:
- just verifying that this URL is for testing purposes only, right?
- will this URL be loaded for everyone that goes to CNX or just those in a CC class?